_posts/2025-01-16-wamr-2024-summary.md

```diff
@@ -0,0 +1,81 @@
+- 292 New Issues: Developers have identified and reported 292 new issues, helping to improve the stability and performance of WAMR.
+- 861 New Stars on GitHub: The project gained 861 new stars, reflecting its growing popularity and recognition.
+- 236 Active Participants: With 236 active participants, the community has been vibrant and engaged, driving WAMR forward with their collective efforts.
+
+Breaking down the contributions further:
+
+- Intel and Others: Half of the PRs, 43.85%, were created by Intel, while the remaining 56.15% were created by the community, including independent contributors and customers integrating WAMR into their products.
+- Community Contributions: The major driving force within the community is company contributors, who provided approximately 85% of the PRs among those created by non-Intel contributors.
+
+The top three non-Intel organized contributors have made significant impacts:
+
+- Midokura: Contributed 33.33% of organized PRs and helped review 35.01% of PRs.
+- Amazon: Contributed 14.33% of organized PRs and helped review 19.90% of PRs.
+- Xiaomi: Contributed 12.40% of organized PRs and helped review 15.11% of PRs.
+
+These contributions have been instrumental in driving WAMR forward, and we extend our heartfelt thanks to everyone involved.
+
+## New Features in WAMR 2024
+
+Several exciting new features have been added to WAMR in 2024, aimed at enhancing the development experience and expanding the capabilities of WAMR. Here are some of the key features:
+
+### Development Tools: Simplifying Wasm Development
+
+One of the most exciting additions to WAMR in 2024 is the introduction of new development tools aimed at simplifying Wasm development. These tools include:
+
+- Linux perf for Wasm Functions: This tool allows developers to profile Wasm functions directly, providing insights into performance bottlenecks.
+- AOT Debugging: Ahead-of-time (AOT) debugging support has been added, making it easier to debug Wasm applications.
+- Call Stack Dumps: Enhanced call stack dumps provide detailed information about the execution flow, aiding in troubleshooting and optimization.
+
+Before these tools, developing a Wasm application or plugin using a host language was a complex task. Mapping Wasm functions back to the source code written in the host language required deep knowledge and was often cumbersome. Debugging information from the runtime and the host language felt like two foreign languages trying to communicate without a translator. These new development tools act as that much-needed translator, bridging the gap and making Wasm development more accessible and efficient.
+
+### Shared Heap: Efficient Memory Sharing
+
+Another significant feature introduced in 2024 is the shared heap. This feature addresses the challenge of sharing memory between the host and Wasm. Traditionally, copying data at the host-Wasm border was inefficient, and existing solutions like externref lacked flexibility and toolchain support.
```

Comment:
Will you mention that this feature is experimental and clarify that the intent is to work towards a standardized approach within the CG? This would help set expectations for early adopters and ensure alignment with the broader Wasm ecosystem. Looking forward to further discussions on this!

Reply:
Sure. I will update the PR soon.